### PR TITLE
Pin .NET 8.0.400 or later 8.0.x SDK

### PR DESCRIPTION
**Pull request by SimonCropp**
SDK 9 is more strict with vulnerability checks.

eg

```
  GraphQL.SystemTextJson.csproj: [NU1904] Warning As Error: Package 'System.Text.Encodings.Web' 4.7.1 has a known critical severity vulnerability, https://github.com/advisories/GHSA-ghhp-997w-qr28
```

this PR forces sdk 8 to unblock people to contribute

### CODE REVIEW

File: src/global.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿{
+  "sdk": {
+    "version": "8.0.401",
+    "allowPrerelease": false,
+    "rollForward": "latestFeature"
+  }
+}
```
Comment (Shane32) [inline src/global.json:7]:
```suggestion
}
```

Comment (Shane32) [reply src/global.json:7]: Can we include the terminating lf ?

File: src/global.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿{
+  "sdk": {
+    "version": "8.0.401",
```
Comment (Shane32) [inline src/global.json:3]:
```suggestion
    "version": "8.0.400",
```
The lastest version of Visual Studio only installs 8.0.400.  Let's use 8.0.400 or a prior version.  (I was running 8.0.302 a few minutes ago.)

**Review by Shane32 — COMMENTED** [review]
I don't usually limit to a specific SDK, but this looks fine, with the rollfoward setting enabled.  :+1:

File: src/global.json (hunk 2 above)
Comment (SimonCropp) [reply src/global.json:3]: 8.0.401 is a security patch

**Comment by SimonCropp** [comment]
not that this is only temporary. the cves will beak the build when the project eventually moves to .net 9

**Comment by Shane32** [comment]
What about if we just do #4030 instead, fixing the problem for both .NET 8 and .NET 9?  Or both PRs?

**Review by Shane32 — APPROVED** [review]

**Pull request merged**